executor/write.go

```diff
@@ -1202,6 +1202,7 @@ type UpdateExec struct {
 
 	SelectExec  Executor
 	OrderedList []*expression.Assignment
+	Ignore      bool
```

Comment:
no need to export `Ignore` ?

Reply:
It needs to export Ignore. In the executor/builder.go file, executorBuilder struct need to set the Ignore field in buildUpdate function